graph/options/options.go

```diff
@@ -227,7 +244,7 @@ func parseAppenders(params url.Values, o Options) []appender.Appender {
 
 // getAccessibleNamespaces returns a Set of all namespaces accessible to the user.
 // The Set is implemented using the map[string]bool convention.
-func getAccessibleNamespaces() map[string]bool {
+func getAccessibleNamespaces() map[string]time.Time {
```

Comment:
The `Options` struct now has a `Namespaces` (type `map[string]NamespaceInfo`) and an `AccessibleNamespaces` (type `map[string]bool`) fields. I need to fill both in the `NewOptions` method. In order to fill the `Namespaces` field I need the timestamps to calculate the duration of each namespace. So, I choose to return the time, instead of booleans which are always `true`. This saves another call to the cluster API to only get timestamps -- And this should answer your other comment.

I can restore the map of bools. But if we want to save calls to the cluster API, this method is the best place to work on; I would still need that `map[string]time.Time` in addition to the `map[string]bool` one. So, this method would return two maps instead of one --- And this way that little additional logic you also commented can go away.

Reply:
Ok... I reverted, but also, as I said, I changed this method to return two maps instead of only one.

Reply:
OK, I understand now why you were assigning timestamps to the accessibleNamespaces.  It can be useful to add an inline comment when performing side-effects/optimizations.  Without the explanation it looked to me like you had added some sort of time logic to AccessibleNamespaces and I was confused.  We can leave it as is but if you want to change it back with a comment that is fine as well.

Reply:
I'll leave it as is for now. I like that it has less loops.